android/native/jni/Android.mk

```diff
@@ -98,6 +98,7 @@ LOCAL_CFLAGS += \
 	-DVERSION_MINOR=${versionMinor} \
 	-DVERSION_PATCH=${versionPatch} \
 	-DVERSION_EXTRA=${versionExtra} \
+	-DDEVELOPMENT_BUILD=${developmentBuild} \
```

Comment:
missing `ENABLE_UPDATE_CHECKER` define _somewhere_ (can also be `config.h` for android)